code/modules/mob/living/simple_mob/subtypes/animal/hyena/hyena.dm

```diff
@@ -58,7 +58,9 @@ They're also cool, and Rykka/Nyria wrote this uwu
 
     speak_emote = list("yaps")
 
-    says_list_type = datum/say_list/hyena
+    say_list_type = datum/say_list/hyena

```

Comment:
Indentation issues here. Do a tab. Oh, wait.... The whole file seems like it's not been done with tabs. Travis won't like that.